chaincode/main.go

```diff
@@ -226,11 +228,45 @@ func formatErrorResponse(err error) peer.Response {
 	}
 }
 
-// main function starts up the chaincode in the container during instantiate
 func main() {
-	// TODO use the same level as the shim or an env variable
-	logger.SetLevel(shim.LogDebug)
-	if err := shim.Start(new(SubstraChaincode)); err != nil {
-		fmt.Printf("Error starting SubstraChaincode chaincode: %s", err)
+
+	logger.SetOutput(os.Stdout)
+	logger.SetLevel(logrus.DebugLevel)
+	logger.Infof("Load TLS certificates")
+
+	key, err := ioutil.ReadFile("/var/hyperledger/tls/client/pair/tls.key")
```

Comment:
maybe add ENV variables to pass certificates, tls keys, etc... here and below hardcoded values could lead to some limitations running the chaincode locally

Reply:
Yes, I will do the update 👍

Reply:
Done !